examples/va/nvidia-mdev/nncp/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../../va/nvidia-mdev/nncp
```

Comment:
Sorry I missed this, since you are not doing anything unique to nncp this will fail to find anything in the directory path listed.  Since we don't need to do anything unique here you can point to the lib path instead: `../../../../lib/nncp`